exercises/SS22/ex_rnw/sol_Ex_1.Rnw

```diff
@@ -0,0 +1,202 @@
+
+\begin{enumerate}
+
+  \item 
+  
+  For the optimal constant model $\fx = \bm{\theta}$ for the loss  $\Lxy=  \big(m(y)-m(\fx)\big)^2,$ we first apply the following substitution $z^{(i)} = m(\yi)$ for each $i=1,\ldots,n,$ and introduce $\bm{\theta}_m = m(\bm{\theta}) \in m(\R).$ 
+%  
+ Note that the inverse of $m$ is continuous and strictly monotone as well, so that the minimizer of the initial optimization problem, i.e.,
+%
+$$
+\min_{f \in \Hspace} \riskef =  \min_{\bm{\theta} \in \R} \sumin ( m(\yi) - m(\bm{\theta}))^2. 
+$$
+%
+is the same as for the ``substituted'' optimization problem, i.e.,
```

Comment:
Das mit dieser Substitution müsste man wohl erklären, sowas haben die meisten bisher noch nie gesehen (und mir darfst du es gerne auch noch mal erklären :) ). Das lohnt sich aber ws. auch in Hinblick auf das SVM-Kapitel.

Reply:
Gerne :-)